### PR TITLE
feat: update chat prompt

### PR DESCRIPTION
**Pull request by darkskygit**
fix AFF-878
fix AFF-852 (maybe)

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#6639** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6639?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/server/src/data/migrations/utils/prompts.ts [hunk 1]
```diff
@@ -17,7 +17,24 @@ export const prompts: Prompt[] = [
...
+      },
+    ],
+  },
+  {
+    name: 'chat:gpt4',
+    model: 'gpt-4-vision-preview',
```
Comment (pengx17) [inline packages/backend/server/src/data/migrations/utils/prompts.ts:30]: Any text input action may contain an image blob, as we can execute actions on multiple blocks where one or more images are present.
Do you think it's possible to implement a logic in the server to intelligently select the most suitable model based on user inputs?

Comment (darkskygit) [reply packages/backend/server/src/data/migrations/utils/prompts.ts:30]: already planned in next stage

**Review by pengx17 — APPROVED** [review]

File: packages/backend/server/src/data/migrations/utils/prompts.ts (hunk 1 above)
Comment (pengx17) [reply packages/backend/server/src/data/migrations/utils/prompts.ts:30]: Ok then. Can you please also make all gpt-4-turbo to vision instead in this PR?

Comment (darkskygit) [reply packages/backend/server/src/data/migrations/utils/prompts.ts:30]: vision may not perform as well as the original version in pure text dialogue and oneshot(action)

Comment (pengx17) [reply packages/backend/server/src/data/migrations/utils/prompts.ts:30]: If it is the case, we need to drop image blocks from inputs of these text actions @donteatfriedrice

**Pull request merged**